src/requirements.yml

```diff
@@ -10,12 +12,15 @@
   name: dev_ssh_access
 - src: https://github.com/cisagov/ansible-role-freeipa-client
   name: freeipa_client
+- src: https://github.com/cisagov/ansible-role-hardening
+  name: harden
 - src: https://github.com/cisagov/ansible-role-htop
   name: htop
 - src: https://github.com/cisagov/ansible-role-nvme
   name: nvme
 - src: https://github.com/cisagov/ansible-role-openvpn
   name: openvpn
+  version: feature/use-ufw-for-nat
```

Comment:
Wasn't this merged almost a month ago in https://github.com/cisagov/ansible-role-openvpn/pull/24?

Reply:
You are eagle-eyed as usual @mcdonnnj! I switched back to using the main branch of `ansible-role-openvpn` with https://github.com/cisagov/openvpn-packer/pull/27/commits/afcc2f1154a03cdaf0b6ad06b74feb15708a8c98.